### PR TITLE
feat: add HasDeviation to building unit syndication response

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/BuildingRegistry.Api.Legacy.Abstractions/Building/Responses/BuildingSyndicationResponse.cs [hunk 1]
```diff
@@ -276,6 +277,12 @@ public class BuildingUnitSyndicationContent
         [DataMember(Name = "IsCompleet", Order = 8)]
         public bool IsComplete { get; set; }
 
+        /// <summary>
+        /// Gebouweenheid afwijking.
+        /// </summary>
+        [DataMember(Name = "AfwijkingVastgesteld", Order = 8)]
```
Comment (jvandaal) [inline src/BuildingRegistry.Api.Legacy.Abstractions/Building/Responses/BuildingSyndicationResponse.cs:283]: I'm not sure about the description in the summary (I took this from the event property summary), nor about AfwijkingVastgesteld.

Comment (jvandaal) [reply src/BuildingRegistry.Api.Legacy.Abstractions/Building/Responses/BuildingSyndicationResponse.cs:283]: Waiting for Business.

**Review by ArneD — APPROVED** [review]
If you have approval from business, lmk.

**Pull request merged**